tests/linux_kernel/helpers/test_waitq.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from drgn import Object
+from drgn.helpers.linux.waitq import (
+    for_each_entry_in_waitqueue,
+    for_each_task_in_waitqueue,
+    waitqueue_active,
+)
+from tests.linux_kernel import LinuxKernelTestCase, skip_unless_have_test_kmod
+
+
+def waiter_tasks(prog):
+    return [
+        prog["drgn_test_waitq_kthread"],
+    ]
+
+
+@skip_unless_have_test_kmod
+class TestWaitqueue(LinuxKernelTestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.waitq = cls.prog["drgn_test_waitq"].address_of_()
+        cls.empty_waitq = cls.prog["drgn_test_empty_waitq"].address_of_()
+        cls.kthread = cls.prog["drgn_test_waitq_kthread"]
```

Comment:
This is unused since you have the `waiter_tasks()` function.

```suggestion
```

Reply:
Done